test/compflags/bradc/help/userhelp.good

```diff
@@ -22,6 +22,7 @@ Optimization Control Options:
       --[no-]dead-code-elimination    Enable [disable] dead code elimination
       --fast                          Use fast default settings
       --[no-]fast-followers           Enable [disable] fast followers
+      --[no-]force-initializers       Enable [disable] default initializers
```

Comment:
Okay, I've got a branch that does this (but it isn't complete yet because I dragged my feet.  I'll have you take a look tomorrow morning when I finish it)